app/models/bus.rb

```diff
@@ -0,0 +1,5 @@
+class Bus < ApplicationRecord
+  validates :bus_name, :bus_type, :bus_brand, :bus_capacity, presence:true
```

Comment:
what do we represent by `bus_capacity`? Will it become 0 when there is no seat left?

Reply:
here `bus_capacity` is the total number of seat a bus have. It's not depend on seat booking.